### PR TITLE
Elasticsearch won't start

### LINKED ISSUE #20: Elasticsearch won't start
I've got bad news 😬 after the recent merges today, Elasticsearch won't start for me. I'm using an Altis v3 stack and after pulling down the fresh Chassis extension it won't start up when provisioning.

I get the following trace from a provision:

```
==> servus-altis.local: Running action triggers before provision ...
==> servus-altis.local: Running trigger...
==> servus-altis.local:
==> servus-altis.local: Running provisioner: shell...
    servus-altis.local: Running: /var/folders/cd/2hthn83x7x189xt7m_jxcqrw0000gn/T/vagrant-shell20200403-79269-8jvj74.sh
==> servus-altis.local: Running provisioner: shell...
    servus-altis.local: Running: inline script
    servus-altis.local: Notice: Compiled catalog for servus-altis.local in environment production in 1.46 seconds
    servus-altis.local: Notice: /Stage[main]/Cavalcade/Service[cavalcade]/ensure: ensure changed 'stopped' to 'running'
    servus-altis.local: Notice: /Stage[main]/Main/Package[git-core]/ensure: created
    servus-altis.local: Notice: /Stage[main]/Chassis::Hosts/Exec[upgrade pip]/returns: executed successfully
    servus-altis.local: Notice: /Stage[main]/Intl/Apt::Ppa[universe]/Exec[add-apt-repository-universe]/returns: executed successfully
    servus-altis.local: Notice: /Stage[main]/Chassis_openssl/File[/vagrant/servus-altis.local.cert]/mode: mode changed '0777' to '0644'
    servus-altis.local: Notice: /Stage[main]/Chassis_openssl/File[/vagrant/servus-altis.local.key]/mode: mode changed '0777' to '0644'
    servus-altis.local: Notice: /Stage[main]/Chassis::Php/Remove_php_version[7.4]/Package[php7.4-common]/ensure: removed
    servus-altis.local: Notice: /Stage[main]/Apt::Update/Exec[apt_update]: Triggered 'refresh' from 1 event
    servus-altis.local: Notice: /Stage[main]/Apcu/Package[php7.2-apcu]/ensure: created
    servus-altis.local: Notice: /Stage[main]/V8js::Extension/Exec[pecl channel-update pecl.php.net]/returns: executed successfully
    servus-altis.local: Notice: /Stage[main]/Chassis::Php/Service[php7.2-fpm]: Triggered 'refresh' from 2 events
    servus-altis.local: Notice: /Stage[main]/Chassis_elasticsearch/Elasticsearch::Instance[es]/Elasticsearch::Service[es]/Elasticsearch::Service::Systemd[es]/Service[elasticsearch-instance-es]/ensure: ensure changed 'stopped' to 'running'
    servus-altis.local: Notice: Failed to connect to the ES instance within timeout window of 30 seconds; giving up.
    servus-altis.local: Error: Failed to apply catalog: Failed to open TCP connection to 0.0.0.0:9200 (Connection refused - connect(2) for "0.0.0.0" port 9200)
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

I had to destroy the machine and re-up for it to work again; provisioning wouldn't clear the issue.

1. What operating system do you use?
Mac OSX 

2. What version of Vagrant are you running?
2.2.7

3. Are you using VirtualBox or VMWare and which version are you using?
Virtualbox

### PR DESCRIPTION
**Pull request by BronsonQuick**
Fixes #20.

@mikeselander Can you please give this a test run?

Heads up to @roborourke as well.

### CODE REVIEW

**Comment by BronsonQuick** [comment]
After losing the last two days trying to fix this we're back to working again. I've tested this with the defaults and the following:
```
elasticsearch:
  repo_version: '6'
  version: '6.8.8'
```
Doing a `curl http://vagrant.local:9200` reveals that things are working as expected! 🙌🏻

**Comment by BronsonQuick** [comment]
@roborourke I couldn't find anything quickly on the `8:` and `11:` prefixes you used for the ES versions but they seemed to break things and not be applied as you'd expect. I only took out the one that was breaking things for now as I thought I'd let you weigh in about that. Taa muchly.

**Comment by roborourke** [comment]
@BronsonQuick I found that info on a thread in the puppet elasticsearch specific to the problem I was seeing with the `-XX:+PrintGCDateStamps` setting. The number prefix for JVM options is documented here: https://www.elastic.co/guide/en/elasticsearch/reference/master/jvm-options.html

I just found what works for Altis v3 but I guess from a clean install only. I have no idea why it'd break an existing install unless it's the different way of installing Java / the switch to the elasticstack repo class. It's a little bit too mysterious if I'm honest... but if you've got it working I'm good with this change.

File: modules/chassis_elasticsearch/manifests/init.pp [hunk 1]
```diff
@@ -56,25 +56,27 @@
...
+      status            => enabled
     }
 
     # Create instances
-    elasticsearch::instance { $options[instances]:
-      config => {
-        'network.host' => '0.0.0.0'
-      },
-    }
+    elasticsearch::instance { $options[instances]:}
```
Comment (roborourke) [inline modules/chassis_elasticsearch/manifests/init.pp]:
```suggestion
    elasticsearch::instance { $options[instances]: }
```

**Review by roborourke — APPROVED** [review]
Small white space thingy to sort out but I'm happy with these changes.

**Comment by roborourke** [comment]
Thanks for working on this by the way - I definitely don't expect you to so it's very very much appreciated.

**Comment by BronsonQuick** [comment]
@roborourke Thanks for the review and the link to the docs! It's weird that the prefix wasn't working. When I eventually found where to look for the Elasticsearch logs it wasn't starting because of the garbage collection. I might have a look again to check if the other options are being applied for the specific versions. For now I'll merge this bad boy so things are working again.

**Pull request merged**